shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java

```diff
@@ -115,7 +117,10 @@ private void checkIfOutputValid(String dateFormatPattern) {
     // Skip checking startTime which relies on system time zone
     String startTime =  CommonUtils.convertMsToDate(1131242343122L, dateFormatPattern);
     List<String> expectedOutput = Arrays.asList("Alluxio cluster summary: ",
-        "    Master Address: testAddress",
+        "    Leader Master Address: testAddress",
+        "    Number of Other Live Masters: 1",
```

Comment:
An Alluxio cluster in fault-tolerant mode only has 3 or 5 masters, unlike workers which can be hundreds or thousands. If the information can be shown obviously in one line, we don't need to use two lines.